sycl/test/basic_tests/device_selector.cpp

```diff
@@ -8,9 +8,11 @@
 
 #include <iostream>
 
-class RejectEverything : public sycl::device_selector {
+using namespace cl::sycl;
+
+class RejectEverything : public device_selector {
 public:
```

Comment:
Why are you changing this file?

Reply:
I did not mean to change this file, but this is a side effect of rebase. 
I will revert this change.